addons/calendar/data/calendar_activity_test_data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="calendar_act_test_meeting" model="mail.activity.type">
```

Comment:
Same as for models: do not define data for tests in standard addons as it means this will be available on production databases. You should instead create it in the setUp of your test class. You can also create an xml ID to use it in your tests. For example
```
cls.activity_type_1 = cls.env['mail.activity.type'].create({
    'name': 'Initial Contact',
    'delay_count': 5,
    'summary': 'ACT 1 : Presentation, barbecue, ... ',
    'res_model_id': cls.env['ir.model']._get('crm.lead').id,
})
cls.env['ir.model.data'].create({
     'name': activity_type.name.lower().replace(' ', '_'),
    'module': 'crm',
    'model': cls.activity_type_1._name,
    'res_id': cls.activity_type_1.id,
})
```